odtp/cli/execution.py

```diff
@@ -107,5 +113,45 @@ def streamlogs(
         sys.exit()
 
 
+@app.command()
+def delete(
+    execution_name: str = typer.Option(
+        None, "--execution-name", help="Specify the name of the execution"
+    ),
+    execution_id: str = typer.Option(
+        None, "--execution-id", help="Specify the ID of the execution"
+    ),
+    project_path: str = typer.Option(
+        None, "--project-path", help="Specify the path for the execution"
+    ),
+    keep_project_path: bool = typer.Option(
+        True, "--keep-project-path", help="Keep the project directory after deleting contents"
+    ),
+):
+    try:
+        if execution_id is None and execution_name is None:
+            raise typer.Exit("Please provide either --execution-name or --execution-id")
+
+        if execution_name:
+            execution_id = db.get_document_id_by_field_value("title", execution_name, db.collection_executions)
+
+        # S3
+        s3_keys = db.get_all_outputs_s3_keys(execution_id)
+        s3M = s3Manager()
+        s3M.deletePaths(s3_keys)
+
+        # DB
+        db.delete_execution(execution_id)
+        
+        # Folders
+        if project_path:
+            odtp_env.delete_folder(project_path, keep_project_path=keep_project_path) 
+
+    except Exception as e:
+        log.error(f"ERROR: Delete execution failed: {e}")       
+        raise typer.Abort()  
+    else:
+        log.info("SUCCESS: execution has been deleted")
```

Comment:
I did not get this message in my tests. I got this instead:
```
odtp/6683af7d5923cfd2e73e613a/
odtp/6683af7d5923cfd2e73e613a/odtp-output.zip
```

Reply:
I tested with the last rebase from develop and it works fine: 

```
03/07/2024 10:42:20 AM - [storage:INFO] 140 storage.py deletePaths - Path 'odtp/66852a00d5b566c9b0c1b4cd' deleted from S3 bucket
03/07/2024 10:42:21 AM - [environment:INFO] 72 environment.py delete_folder - Folder deleted: /Users/carlosvivarrios/pro/ODTP/odtp-instance/dts/dt-example/dt-example/execution
03/07/2024 10:42:21 AM - [execution:INFO] 165 execution.py delete - SUCCESS: execution has been deleted
SUCCESS: execution has been deleted
```